DirectEmployers/MyJobs
# Pd 3566 commm action button feature branch

## What's this PR do?
Add a common action button in PRM/NUO/PA , TJD, and Manage Users.
## Screenshots (if appropriate)

## Where should the reviewer start?

## How should this be manually tested?
* PRM, without selecting a partner, click the "Add Record" action button. The "Edit Record" form should appear with only the Partner field. select a partner and the rest of the form should appear.
* PRM, Select a partner, then click the "Add Record" button. All fields should be visible with the Partner preselected.
* PRM, the "New Record" button functionality shouldn't be affected.
* TJD, the action button should direct to "New organization"
* Manage user, the action button should direct to "Add User"


## Any background context you want to provide?

## PR Checklist

### Front-End


### Separation of Concerns


### Tests


### Migrations
